system/src/system_network_manager.cpp

```diff
@@ -577,6 +586,14 @@ void NetworkManager::handleIfPowerState(if_t iface, const struct if_event* ev) {
     }
 }
 
+void NetworkManager::handleIfPhyState(if_t iface, const struct if_event* ev) {
+    if (networkStatus_ == NetworkStatus::NETWORK_STATUS_POWERING_ON && ev->ev_phy_state->state == IF_PHY_STATE_ON) {
```

Comment:
Let's add a FIXME that this currently works somewhat properly only if there is just 1 network interface.